executor/aggregate.go

```diff
@@ -174,6 +174,9 @@ type HashAggExec struct {
 	prepared         bool
 	executed         bool
 
+	isAllFirstRow     bool
+	firstRowProcessed int
```

Comment:
Please reset firstRowProcessed to 0 in function (e *HashAggExec) Close(). 
Because Apply executor maybe Open()/Close() an executor many times, the wrong firstRowProcessed maybe lead a wrong result.